muselog/__init__.py

```diff
@@ -76,8 +76,5 @@ def setup_logging(root_log_level: Optional[str] = None,
 
     # Log to stdout if enabled
     if "ENABLE_STDOUT_LOG" in os.environ and os.environ["ENABLE_STDOUT_LOG"] == "True":
```

Comment:
I think we'll need to rethink this design. ENABLE_STDOUT_LOG doesn't make sense here, since we are just doing formatting. We'll want to combine this formatter with the existing console handler functionality, but without disturbing the existing functionality or extending it in a unnatural way. Any ideas? :)